examples/cats.js

```diff
@@ -1,6 +1,5 @@
 
-process.env.CYCLIC_DB = 'glamorous-battledress-tickCyclicDB'
-const db = require('../src/')
+const db = require('../src/')('glamorous-battledress-tickCyclicDB')
```

Comment:
Do we want to also allow for named options params?

`{tableName: 'glamorous-battledress-tickCyclicDB'}`